dexcom/event.go

```diff
@@ -196,44 +217,69 @@ func (e *Event) Validate(validator structure.Validator) {
 			e.validateHealth(validator)
 		case EventTypeInsulin:
 			e.validateInsulin(validator)
+		case EventTypeNotes:
+			e.validateNote(validator)
+		case EventTypeBG:
+			e.validateBG(validator)
+		case EventTypeUnknown:
+			e.validateUnkown(validator)
```

Comment:
Nit: typo in method name `e.validateUnkown(validator)` - but matches the actual method declaration :)

Reply:
yeah - that will be me. Spelling is not a string point :)